simulator/src/simulator/utils/accounts.ts

```diff
@@ -57,3 +57,33 @@ export async function getDefaultAccount(node: SimulationNode): Promise<string> {
 
   return resp.content.account.name
 }
+
+/**
+ * Sets the default account on a node.
+ *
+ * @param node The node to set the default account on
+ * @param account The account to set as the default account
+ * @throws Error if the account does not exist
+ */
+export async function setDefaultAccount(node: SimulationNode, account: string): Promise<void> {
+  await node.client.useAccount({ account })
+}
+
+/**
+ * Imports an account on a node. This is done via `wallet:import`, so the account must be either the
+ * copy-pasted output of `wallet:export` or a raw spending key.
+ *
+ * @param node The node to import the account on
+ * @param account The account to import, in the form of a string blob
+ * @param rescan Whether to explicitly rescan the blockchain for transactions involving the account
+ */
+export async function importAccount(
+  node: SimulationNode,
+  account: string,
+  rescan?: boolean,
+): Promise<void> {
+  await node.executeCliCommandAsync('wallet:import', [account])
```

Comment:
Can we just use the RPC command for this? If we store the account as spendKey, publicKey etc then we can just use the RPC command to import

Reply:
The genesis account is stored as a string blob right now, is there an easy way to go from string -> {spendkey, publickey} without re-writing a lot of the logic in `wallet:import`?

Reply:
Since this is supposed to basically take any account that comes out of wallet:export, I'm okay leaving this as-is. I think there's a general discussion to be had around moving our account parsing logic into the SDK though